grc/core/generator/cpp_top_block.py

```diff
@@ -222,9 +222,9 @@ def _blocks(self):
         fg = self._flow_graph
         parameters = fg.get_parameters()
 
-        # List of blocks not including variables and imports and parameters and disabled
+        # List of blocks not including variables and parameters and disabled
         def _get_block_sort_text(block):
-            code = block.cpp_templates.render('make').replace(block.name, ' ')
+            code = block.cpp_templates.render('declarations')
```

Comment:
Are we sure this does not break something else?

Reply:
This change only affects the order of generating the cpp code. Without this change sometimes the order how cpp code is generated was wrong.